tests/unit/flow.test.js

```diff
@@ -161,19 +153,6 @@ describe('Flow', () => {
         expect(setStateSpy).toBeCalled();
         expect(historySpy.push).toHaveBeenCalledWith('repair-kitchen-cupboard-problems');
       });
-      test('does not redirect to repair-kitchen-cupboard-problems when repairProblemBestDescription value is random', ()=>{
-        const state = {
-          prevStep: 'repair-description',
-          step: 'repair-description',
-          data: {
-            repairProblemBestDescription: {
-              value: 'random'
-            }
-          }
-        }
-        flow.prevStep(state);
-        expect(historySpy.push).not.toHaveBeenCalledWith('repair-kitchen-cupboard-problems');
-      });
```

Comment:
Note: these tests are now redundant as the back location is determined by the PrevSteps array